src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -87,7 +87,10 @@ export default class FrameComponent extends Component<FrameComponentProps> {
     frame: Frame,
     selectedFrame: Frame
   ) {
-    if (e.nativeEvent.which == 3 && selectedFrame.id != frame.id) {
+    if (
+      (e.nativeEvent.which == 3 && selectedFrame.id != frame.id) ||
+      selectedFrame.id === frame.id
+    ) {
```

Comment:
i feel like we might want: `e.nativeEvent.which == 3 || selectedFrame.id === frame.id`

Reply:
true